proto/neutron/contractmanager/query.proto

```diff
@@ -18,7 +18,7 @@ service Query {
   }
 
   // Queries a Failure by contract address and failure ID.
-  rpc AddressFailure(QueryFailuresRequest) returns (QueryFailuresResponse) {
+  rpc AddressFailure(QueryFailureRequest) returns (QueryFailuresResponse) {
```

Comment:
How about making `QueryFailureResponse` struct without pagination? It seems strange that we return it in array even tho it's always one failure. And pagination always null